pages/index.js

```diff
@@ -0,0 +1,11 @@
+import Head from 'next/head'
+import styles from '../styles/Home.module.css'
+import UsMap from '../components/UsMap'
+
+export default function Home() {
+  return (
+    <div>
+      <UsMap></UsMap>
```

Comment:
NIT: since we only have one component, let's just return that - remove <div>

Reply:
Unless there is a reason it's nested here of course

Reply:
Roger - no reason I can think of other than that div was there from the scaffold so I used it